### DISCUSSION TITLE
Why would a QTable column get recursive?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#13129] [General - Components / Directives / etc] [answered]
In a QTable I have several columns declared, and one of them is giving me troubles:

```
const columns = [
  {name: 'author', field: (row: Case) => row.Author, label: 'Author', align: 'left', sortable: true},
  {name: 'assignedTo', field: (row: Case) => row.AssignedTo, label: 'Assigned To', align: 'left', sortable: false},
]
```

`row.Author`  is a string, and `row.AssignedTo` is an Array of strings. This gives a warning in the console and the table displays fine (but the laptop is noisy and the page is slow)

>  `[Vue warn]: Maximum recursive updates exceeded in component <QTable>. This means you have a reactive effect that is mutating its own dependencies and thus recursively triggering itself. Possible sources include component template, render function, updated hook or watcher source function.`

Removing the line fixes the problem.

Changing the line to 

    {name: 'assignedTo', field: (row: Case) => row.AssignedTo?.join(' '), label: 'Assigned To', align: 'left', sortable: false},

also solves the issue.

It would seem that there is some operation on the array `row.AssignedTo` that triggers (re-)rendering in a loop.

**Comment by metalsadman** [comment, marked as answer]
because `field` correspond to a property or a function which QTable expects to resolve in a plain value so that it can be displayed thru assignment in an internal property in cols array called ` value`., hence why joining it worked when you joined it, the value is also used for stuff like filtering etc...
